pages/_profiles/DefinedTerm/0.1-DRAFT.html

```diff
@@ -0,0 +1,119 @@
+---
+redirect_from:
+- "/profiles/DefinedTerm/"
+- "/profiles/DefinedTerm"
+# spec_info content generated using GOWeb
+# DO NOT MANUALLY EDIT THE CONTENT
+cross_walk_url: ''
+dde_ui_url: https://discovery.biothings.io/view/bioschemasdrafts/bioschemasdrafts:DefinedTerm
+gh_tasks: https://github.com/Bioschemas/specifications/labels/type%3A%20DefinedTerm
+group: ''
+hierarchy:
+- Thing
+- Intangible
+- DefinedTerm
+json-ld_url: DefinedTerm/jsonld/DefinedTerm_v0.1-DRAFT.json
+live_deploy: /liveDeploys
+mapping:
+- bsc_description: ''
+  cardinality: ONE
+  controlled_vocab: ''
+  description: A [[DefinedTermSet]] that contains this term.
+  example: |-
+    {
+     "@type": "DefinedTerm",
+     "inDefinedTermSet": "http://purl.obolibrary.org/obo/ncbitaxon.owl",
```

Comment:
If the cardinality is ONE, the example should not show two.
I am adding your examples to the other PR because yours give me and error. Only the firs URL will be used.

Reply:
Ah, misunderstanding on my side, I gave two alternative examples. We can use any one of them.